src/GraphQl/Subscription/MercureSubscriptionIriGeneratorInterface.php

```diff
@@ -24,5 +24,5 @@ interface MercureSubscriptionIriGeneratorInterface
 {
     public function generateTopicIri(string $subscriptionId): string;
 
-    public function generateMercureUrl(string $subscriptionId): string;
+    public function generateMercureUrl(string $subscriptionId, ?string $hub = null): string;
```

Comment:
bc allowed because interface is experimental, we could even enforce this argument?

Reply:
If `$hub` is `null`, the registry will retrieve the default hub.
Therefore IMO this argument must be optional anyway.